### PR TITLE
getting data from data base

### PR DESCRIPTION
**Pull request by daleel1**
Adding proxy 
Getting Data from database

### CODE REVIEW

File: backend/route/router.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 const express = require("express");
+const app = express();
```
Comment (anton-kozyr) [inline backend/route/router.js]: Can be removed
```suggestion
```

File: backend/route/router.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+
 const router = express.Router();
+app.use(router);
```
Comment (anton-kozyr) [inline backend/route/router.js]: Can be removed
```suggestion
```

**Review by anton-kozyr — CHANGES_REQUESTED** [review]
```app``` object is not required and can be removed

**Review by anton-kozyr — APPROVED** [review]

File: backend/route/router.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ router.post("/", (req, response) => {
       console.log(err.stack);
```
Comment (dahfool) [inline backend/route/router.js]: @reber12 Please remove console.log.

**Review by dahfool — APPROVED** [review]

**Pull request merged**